ui/ducks/locale/locale.test.ts

```diff
@@ -0,0 +1,15 @@
+import locales from '../../../app/_locales/index.json';
+import { getIntlLocale } from './locale';
+
+const createMockStateWithLocale = (locale: string) => ({
+  localeMessages: { currentLocale: locale },
+});
+
+describe('getIntlLocale', () => {
+  it.each(locales)('can convert locale "%s" to BCP 47 format', (locale) => {
```

Comment:
Love this test! But if this is the happy path, should we also have a unit test to verify we throw if a bad locale is in the state?

Reply:
Good call. Done.